new/LIT6361MEMiracle.xml

```diff
@@ -0,0 +1,85 @@
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Miracle of ʾEwosṭātewos: on the pious monk who was unjustly imprisoned and was rescued by the saint's appearance</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MV"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Miracle"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="MV" when="2021-01-20">Created entity</change>
+            <change who="MV" when="2021-01-21">Fixed title</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listBibl type="secondary">
+                    <bibl>
+                        <ptr target="bm:EMML5"/>
+                        <citedRange unit="page">131</citedRange>
+                    </bibl>
+                </listBibl>
+                <listRelation>
+                    <relation name="saws:formsPartOf" active="LIT6361MEMiracle" passive="LIT4315Miracles"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <note>This incipit is taken from <ref type="mss" corresp="BGV009"/>.</note>
+                    <ab>
+                        ፲ወ፪። ተአምሪሁ፡ ለአቡነ፡ ክቡር፡ ማር፡ ኤዎስጣቴዎስ፡ ነቢይ፡ ወሐዋርያ፡ ድንግል፡ ወካህን። <gap reason="ellipsis" resp="MV"/>                 
+                        ወካዕበ፡ ተብህለ፡ በእንተ፡ ፩ መነኮስ፡ ዘኮነ፡ ያፈቅሮ፡ ለአቡነ፡ ኤዎስጣቴዎስ፡ ስዩመ፡ ውእቱ፡ ላዕለ፡ ደብሩ፡ ወሀለዉ፡ መነኮሳት፡ 
```

Comment:
፩ መነኮስ፡ = space between the number and the name

Reply:
I think visualization is misleading: it's just one space between ፩ and መነኮስ፡. Anyway, one or more than one is indifferent: it will be visualized as one in the printed/digital output.

Reply:
I see one space, but what I meant is: is there a space at all in the ms. (when the number is not followed by the word division mark I think should be attached to the following word)? Do we need to add a space between the number and the name which follows?

Reply:
This is a very interesting question. I implicitly inherited a practice used in the Bodleian MSS project, that is to leave one space between numeral and word. This seems to facilitate filtered research (@SusanneHummel ). But you are right: we have never discussed this question in our project and we need to. We might briefly do it in the next meeting on Thursday.